README.md

```diff
@@ -144,7 +145,7 @@ nnoremap <leader>tl :<c-u>exec v:count.'Tclear'<cr>
 ### Troubleshooting
 
 Most standard file extensions for the above REPLs are picked up by Neovim/Vim's
-default filetype plugins. However, there are two exceptions:
+default filetype plugins. However, there are three exceptions:
```

Comment:
To avoid having to update this when the list grows, what do you think to change it tO

```suggestion
default filetype plugins. However, there are some exceptions:
```